core/src/main/java/org/frankframework/http/cxf/SOAPProviderBase.java

```diff
@@ -241,6 +227,31 @@ public SOAPMessage invoke(SOAPMessage request) {
 		}
 	}
 
+	private void handleIncomingAttachments(Iterator<AttachmentPart> attachmentParts, PipeLineSession pipelineSession) {
```

Comment:
Ik zou hier graag in een comment of in de method-naam willen zien dat deze method alleen is voor backwards compatibility met oude FF versies.